Box.V2/Models/Request/BoxRetentionPolicyRequest.cs

```diff
@@ -55,5 +55,12 @@ public class BoxRetentionPolicyRequest
         /// </summary>
         [JsonProperty(PropertyName = "custom_notification_recipients")]
         public List<BoxRequestEntity> CustomNotificationRecipients { get; set; }
+
+        /// <summary>
+        /// Used to determine the type of retention policy, value can be modifiable or non-modifiable
+        /// When updating a retention policy, you can use non-modifiable type only. You can convert a modifiable policy to non-modifiable, but not the other way around.
+        /// </summary>
+        [JsonProperty(PropertyName = "retention_type")]
+        public string RetentionType { get; set; }
```

Comment:
I think that we can use enum there if we have only 2 possible values.

Reply:
I can only add it into `Box.V2/Config/Constants.cs`, it can not be added into enum due to the name `modifiable` and `non-modifiable`.

Reply:
I believe that you still should be able to express this as an enum using `[EnumMember(Value = "non-modifiable)] `attribute. See `BoxSignRequestStatus` for example